pkg/services/rpcsrv/server_test.go

```diff
@@ -1984,48 +2181,54 @@ func testRPCProtocol(t *testing.T, doRPCCall func(string, string, *testing.T) []
 		require.Equal(t, trigger.PostPersist, res.Executions[1].Trigger)
 		require.Equal(t, vmstate.Halt, res.Executions[1].VMState)
 	})
-
-	t.Run("submit", func(t *testing.T) {
+	t.Run("submitblock", func(t *testing.T) {
```

Comment:
Let's keep the old test name (`submit`) and convert some of the tests from `submitblock` to `sendrawtransaction` (those tests that are transaction-related).

Reply:
Could you please specify tests to convert to `sendrawtransaction` function? From my point of view, if I can get an error with `submitblock` it means that it is a suitable test. So I wish to understand logic here.

Reply:
We have a set of transaction-specific errors that are returned from `bc.verifyAndPoolTx`. It's true that you can get the same errors while submitting the block, but at the same time the source of these errors is invalid transaction. Thus, I considered it to be more intuitive to use `sendrawtransaction` for these cases. But you're right as far, let's keep it as is.